src/components/Navbar/Navbar.css

```diff
@@ -1,75 +1,79 @@
 /* Position and sizing of burger button */
 .bm-burger-button {
-    position: fixed;
-    width: 36px;
-    height: 30px;
-    left: 36px;
-    top: 36px;
-  }
-  
-  /* Color/shape of burger icon bars */
-  .bm-burger-bars {
-    background: white;
-    transition: background .2s ease;
-  }
-  
-  /* Color/shape of burger icon bars on hover*/
-  .bm-burger-bars-hover {
-    background: rgba(255, 255, 255, 0.708);
-  }
-  
-  /* Position and sizing of clickable cross button */
-  .bm-cross-button {
-    height: 500px;
-    width: 500px;
-  }
-  
-  /* Color/shape of close button cross */
-  .bm-cross {
-    background: white;
-  }
-  
-  /*
+  position: fixed;
+  width: 36px;
+  height: 30px;
+  left: 36px;
+  top: 36px;
+}
+
+/* Color/shape of burger icon bars */
+.bm-burger-bars {
+  background: white;
+  transition: background 0.2s ease;
+}
+
+/* Color/shape of burger icon bars on hover*/
+.bm-burger-bars-hover {
+  background: rgba(255, 255, 255, 0.708);
+}
+
+/* Position and sizing of clickable cross button */
```

Comment:
It looks like a lot of these changes are coming from Prettier not having been applied. It might be a good idea for everyone to:

1. Get themselves on the latest version of Node and npm
2. Re-run `npm i` in this project
3. Re-run `prettier --write` _(or whichever command this project can use to re-Prettify all files, I forget)
4. Have one person send a PR with any changes from steps 2 or 3

It's hard to review changes / look over your own work when there's this much noise in the diffs.